modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -320,6 +320,23 @@ void GDScriptAnalyzer::get_class_node_current_scope_classes(GDScriptParser::Clas
 	}
 }
 
+#ifdef DEBUG_ENABLED
+void GDScriptAnalyzer::check_access_private_member(GDScriptParser::IdentifierNode *p_identifier, const bool p_is_call) {
+	if (p_identifier == nullptr) {
+		return;
+	}
+	if (!String(p_identifier->name).begins_with("_")) {
+		return;
+	}
+	if (parser->current_class->get_datatype().kind != GDScriptParser::DataType::CLASS && parser->current_class->get_datatype().kind != GDScriptParser::DataType::SCRIPT) {
+		return;
+	}
+	if (!parser->current_class->has_member(p_identifier->name)) {
```

Comment:
Missing a `parser->current_class == nullptr` check. This might happen sometimes in editor when the script is invalid at the moment.